src/app/_variables.scss

```diff
@@ -43,6 +43,9 @@ $gilded-triumph-border-color: #a78355;
 // Yellow used for enhanced weapon perks
 $enhancedYellow: #f3cf55;
 
+// The in-game color for the shaped weapon icon
+$shaped: #d25336;
```

Comment:
This could possibly be combined with `$deepsight-border-color` which shares the same value.